src/main/java/androidx/view/ViewGroup.kt

```diff
@@ -77,6 +77,21 @@ val ViewGroup.children: Sequence<View>
         override fun iterator() = this@children.iterator()
     }
 
+/**
+ * Executes [block] with a typed version of the ViewGroup's layoutParams and reassigns the
+ * layoutParams with the updated version.
+ *
+ * @see ViewGroup.getLayoutParams
+ * @see ViewGroup.setLayoutParams
+ **/
+inline fun <reified T> ViewGroup.layoutParamsAs(
```

Comment:
The name does not match the behavior. I would expect this method to simply be a get+cast, not to trigger a requestLayout via setting the LPs.

Reply:
Do you think `withLayoutParams` would be better? Suggestions?

Reply:
`updateLayoutParams`?

Reply:
@romainguy would `updateLayoutParams` or `updateLayoutParamsAs` work for you?